modules/dxp/apps/segments/segments-asah-connector/src/main/java/com/liferay/segments/asah/connector/internal/expression/FilterByCountIndividualSegmentsExpressionVisitorImpl.java

```diff
@@ -112,6 +114,61 @@ else if (Objects.equals("day", value)) {
 			"Unsupported operation with value " + value);
 	}
 
+	@Override
+	public FilterByCount.Day visitFunctionCallExpression(
+		@NotNull
+			IndividualSegmentsExpressionParser.FunctionCallExpressionContext
+				functionCallExpressionContext) {
+
+		ParseTree parseTree1 = functionCallExpressionContext.getChild(0);
+
+		Object functionName = parseTree1.accept(this);
+
+		if (Objects.equals("between", functionName)) {
+			ParseTree parseTree2 = functionCallExpressionContext.getChild(2);
+
+			List<String> parameters = (List<String>)parseTree2.accept(this);
+
+			if (parameters.size() != 3) {
+				throw new UnsupportedOperationException(
+					StringBundler.concat(
+						"Unsupported function BETWEEN ",
+						String.valueOf(parameters.size()), " params"));
+			}
+
+			if (!Objects.equals("day", parameters.get(0))) {
+				throw new UnsupportedOperationException(
+					"Unsupported function BETWEEN with first param " +
+						String.valueOf(parameters.get(0)));
+			}
+
+			return new FilterByCount.Day(
+				StringUtil.lowerCase(Function.BETWEEN.toString()),
```

Comment:
Maybe we can use the _normalizeOperator function here?

Reply:
why if only use this one time ? It is a good practice in Liferay to do that?

Reply:
We already have this function in the code: 

private String _normalizeOperator(Token token) {
		return _normalizeStringLiteral(StringUtil.toLowerCase(token.getText()));
	}

Reply:
I am not able to see that function. Where is it? Sorry